packages/canary-integration/src/__tests__/model-instance-document.test.ts

```diff
@@ -112,27 +180,34 @@ describe('ModelInstanceDocument API http-client tests', () => {
   })
 
   test('Create and update doc', async () => {
-    const doc = await ModelInstanceDocument.create(
-      ceramic,
-      CONTENT0,
-      midMetadata
-    )
+    const doc = await ModelInstanceDocument.create(ceramic, CONTENT0, midMetadata)
     expect(doc.content).toEqual(CONTENT0)
 
+    // confirm stream was added to index
```

Comment:
Dev artefact to remove, maybe?

Reply:
reverted back to original state as discussed for future canary test.